### PR TITLE
[MODULAR] Ashwalker Re-buff

### PR DESCRIPTION
**Pull request by Vizoplez**
## About The Pull Request
Re-adds Ashwalker NV and Ash Resistance, along with adjusting their damage values once more.

## Why It's Good For The Game
Used to be on the oldbase, and with the lack of content for ashwalkers right now, gives them flexability.

## Changelog
:cl:
add: Ashwalker immunity & NV & damage resists
/:cl:

### CODE REVIEW

File: modular_skyrat/modules/ashwalkers/Ashwalker/Ashwalkers.dm [hunk 1]
```diff
@@ -0,0 +1,3 @@
+/datum/species/lizard/ashwalker
+	mutanteyes = /obj/item/organ/eyes/night_vision
+	weather_immunities = list("ash")
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/ashwalkers/Ashwalker/Ashwalkers.dm]: You dare come into these lands without the proper newlines? Begone thot!

Comment (Vizoplez) [reply modular_skyrat/modules/ashwalkers/Ashwalker/Ashwalkers.dm]: A

**Review by Gandalf2k15 — APPROVED** [review]
Seems fine, I assume you've tested it.

**Comment by Vizoplez** [comment]
> Seems fine, I assume you've tested it.

These use the same values as oldbase, and use the same code as free golem along w/ similar overrides, I cant build TGUI right now, but yes, it should compile and work

**Pull request merged**